redaxo/src/addons/structure/plugins/content/lang/de_de.lang

```diff
@@ -59,8 +59,8 @@ content_toarticle_failed = Diese Kategorie konnte nicht in einen Artikel umgewan
 content_toarticle_ok = Diese Kategorie wurde erfolgreich in einen Artikel umgewandelt
 move_category = nach
 category_moved = Kategorie wurde verschoben
-content_types = Bereiche (ctypes)
-content_type = Bereich (ctype)
+content_types = Bereiche (ctypes) und Module
+content_type = Bereich (ctype) und Module
```

Comment:
Das ist ja die Überschrift über jedem Bereich (mit ID hinten dran), ich glaube das würde ich auf "Bereich (ctype) ID=1" belassen.

Die Umbenennung im Tab finde ich ok.
Am besten zumindest in Englisch auch anpassen.
Bei den anderen Sprachen ist wieder die Frage, ob wir den Key löschen und neu übersetzen lassen, oder ob uns die Änderung zu unwichtig ist, und wir lieber die vorhandene Übersetzung belassen.

Reply:
Die Umbenennung im Tab finde ich auch ok. Nur sollte der Hinweis dann nicht eher etwas in Richtung "Modulzuweisung" oder ähnliches heißen?

Reply:
- erlaubte Module 
- zugelassene Module 
- verfügbare Module

Reply:
"verfügbare" finde ich nicht gut. Ansonsten ist es mir egal (auch Thomas "Modulzuweisung" wäre für mich ok).